libs/net/dal/Services/ReportService.cs

```diff
@@ -851,9 +851,9 @@ public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = nul
 
                 // Modify the query to exclude content.
                 var excludeOnlyTheseContentIds = excludeContentIds.Any() && !sectionSettings.OverrideExcludeHistorical ? excludeContentIds : Array.Empty<long>();
-                var excludeAboveAndHistorical = sectionSettings.RemoveDuplicates
+                var excludeAboveAndHistorical = sectionSettings.OverrideExcludeHistorical
```

Comment:
If a user selects the "Include all content from linked folder even if in prior report" in a section does this work?  This is where the `overrideExcludeHistorical` setting comes from.  It's a per-section configuration option.  It ensures that that specific section will still include content that was also in a linked report.  The reason for this is because Scott wants the folder content to show up no matter what.

![image](https://github.com/user-attachments/assets/87292a92-659f-435d-aff8-f19b12ddfeff)

Reply:
It would, because the variable excludeOnlyTheseContentIds on line 853 is assigned like that:
var excludeOnlyTheseContentIds = excludeContentIds.Any() && !sectionSettings.OverrideExcludeHistorical ? excludeContentIds : Array.Empty<long>();
It would only be populated if the Override is checked.
In this case excludeOnlyTheseContentIds is empty and would not exclude those contents.